src/NuGet.Core/NuGet.PackageManagement/BuildIntegratedPackageReference.cs

```diff
@@ -40,6 +41,27 @@ public BuildIntegratedPackageReference(LibraryDependency dependency, NuGetFramew
             Dependency = dependency;
         }
 
+        /// <summary>
+        /// Create a PackageReference based on a LibraryDependency.
+        /// </summary>
+        /// <param name="dependency">Full PackageReference metadata.</param>
+        /// <param name="installedVersion">Package installed.</param>
+        public BuildIntegratedPackageReference(LibraryDependency dependency, NuGetFramework projectFramework, PackageIdentity installedVersion)
```

Comment:
nitpick: It's common when there are multiple constructor overloads, for only one of them to have a method body and the others call the one "real" constructor. That makes it easier to understand what the difference is between the constructors, and minimise risk of bugs when new properties or fields are added to the class, and trying to make sure that all of the constructors set the new members. There are no new members here in your change, but it's more of an FYI. I think it would have been nice for the existing constructor to call this one, to show that they really do the same thing. When different ctor overloads call different base ctors, that's a good scenario to not have all ctors call a single ctor, but I don't think that's happening here.